src/engine/Loader.ts

```diff
@@ -120,6 +230,24 @@ export class Loader extends Class implements ILoader {
     return this._numLoaded === this._resourceCount;
   }
 
+  /**
+   * Shows the play button and returns a promise that resolves when clicked
+   */
+  public showPlayButton(): Promise<any> {
+    this._playButton.style.display = 'block';
+    let promise = new Promise();
+    this._playButton.onclick = () => promise.resolve();
+
+    if (this.suppressPlayButton) {
```

Comment:
Correct me if I'm wrong but if this flag is `true`, it will cause a pending Promise to be laying around in memory on L238, right? Should this have an `else` block with the rest of this code in it?